.gitignore

```diff
@@ -3,3 +3,4 @@ image.tar
 scripts/*.js
 .DS_Store
 .vscode/
+docker-images/
```

Comment:
see the red circle?

Reply:
I do not see a red circle

Reply:
but I know what you are talking about - I've added a space at the end